.github/workflows/prototype-tests.yml

```diff
@@ -0,0 +1,44 @@
+name: tests
+
+on:
+  pull_request:
+
+jobs:
+  prototype:
+    strategy:
+      matrix:
+        os:
+          - ubuntu-latest
+          - windows-latest
+          - macos-latest
+      fail-fast: false
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - name: Set up python
+        uses: actions/setup-python@v3
+        with:
+          python-version: 3.7
+
+      - name: Upgrade system packages
+        run: python -m pip install --upgrade pip setuptools wheel
+
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Install PyTorch nightly builds
+        run: pip install --progress-bar=off --pre torch torchdata --extra-index-url https://download.pytorch.org/whl/nightly/cpu/
+
+      - name: Install torchvision
+        run: pip install --progress-bar=off --no-build-isolation --editable .
+
+      - name: Install other prototype dependencies
+        run: pip install --progress-bar=off scipy pycocotools h5py iopath
+
+      - name: Install test requirements
+        run: pip install --progress-bar=off pytest pytest-mock
+
+      - name: Run prototype tests
+        shell: bash
+        run: pytest --durations=20 test/test_prototype_*.py
```

Comment:
@pmeier let's add `-vvv` flag to see more info on failing tests ?

Reply:
AFAIK, the extra verbosity only applies to the output during the run. The information displayed by `pytest` for failing tests in teh report should be the same. Do you have a counter example?

Reply:
https://github.com/pytorch/vision/runs/6254487636?check_suite_focus=true#step:9:195

Reply:
The report is there:

```

================================== FAILURES ===================================
___ test_correctness_perspective_bounding_box[startpoints2-endpoints2-cpu] ____
Traceback (most recent call last):
  File "D:\a\vision\vision\test\test_prototype_transforms_functional.py", line 1210, in test_correctness_perspective_bounding_box
    torch.testing.assert_close(output_bboxes, expected_bboxes)
  File "c:\hostedtoolcache\windows\python\3.7.9\x64\lib\site-packages\torch\testing\_comparison.py", line 1321, in assert_close
    msg=msg,
  File "c:\hostedtoolcache\windows\python\3.7.9\x64\lib\site-packages\torch\testing\_comparison.py", line 1074, in assert_equal
    raise error_metas[0].to_error()
AssertionError: Tensor-likes are not close!

Mismatched elements: 1 / 16 (6.2%)
Greatest absolute difference: 1.4543533325195312e-05 at index (0, 2) (up to 1e-05 allowed)
Greatest relative difference: 1.1173174124760395e-05 at index (0, 2) (up to 1.3e-06 allowed)
```

Is this not what you are looking for?